src/libs/DistanceRequestUtils.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'underscore';
+import CONST from '../CONST';
+
+const getDefaultMileageRate = (policy) => {
+    if (!policy || !policy.customUnits) {
+        return null;
+    }
+
+    const distanceUnit = _.find(_.values(policy.customUnits), (unit) => unit.name === CONST.CUSTOM_UNITS.NAME_DISTANCE);
+    if (!distanceUnit) {
+        return null;
+    }
+
+    const distanceRate = _.find(_.values(distanceUnit.rates), (rate) => rate.name === CONST.CUSTOM_UNITS.DEFAULT_RATE);
+    if (!distanceRate) {
+        return null;
+    }
+
+    return {
+        rate: distanceRate.rate,
+        unit: distanceUnit.attributes.unit,
+    };
+};
+
+function convertDistanceUnit(distanceInMeters, unit) {
+    if (typeof distanceInMeters !== 'number' || (unit !== CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES && unit !== CONST.CUSTOM_UNITS.DISTANCE_UNIT_KILOMETERS)) {
+        throw new Error('Invalid input');
```

Comment:
How would it be possible for this to ever throw? It's not based on user input, so can we be pretty confident that all other code are passing the params properly?

Reply:
https://github.com/Expensify/App/pull/25707/#discussion_r1310640056 😅